src/brsTypes/components/RoAssociativeArray.ts

```diff
@@ -223,8 +258,22 @@ export class RoAssociativeArray extends BrsComponent implements BrsValue, BrsIte
         },
     });
 
-    /** Given a key, returns the value associated with that key. This method is case insensitive. */
+    /** Given a key, returns the value associated with that key.
+     * This method is case insensitive either-or case sensitive, depends on whether `setModeCasesensitive` was called or not.
+     */
     private lookup = new Callable("lookup", {
+        signature: {
+            args: [new StdlibArgument("key", ValueKind.String)],
+            returns: ValueKind.Dynamic,
+        },
+        impl: (interpreter: Interpreter, key: BrsString) => {
+            let lKey = key.value;
```

Comment:
Perhaps I've missed something — why's this variable `lKey` while the rest are `key`?

Reply:
idk why, but it was as is before my changes

Reply:
fixed